libbpf-tools/biolatency.bpf.c

```diff
@@ -6,9 +6,14 @@
 #include <bpf/bpf_tracing.h>
 #include "biolatency.h"
 #include "bits.bpf.h"
+#include <linux/version.h>
 
 #define MAX_ENTRIES	10240
 
+#define KERNEL_VERSION(a, b, c) (((a) << 16) + ((b) << 8) + (c))
```

Comment:
we should be able to remove this, once latest libbpf sync gets through and libbpf is updated in BCC repo. bpf_helpers.h is not providing this macro